google-cloud-storage/src/main/java/com/google/cloud/storage/BlobInfo.java

```diff
@@ -276,6 +277,8 @@ static CustomerEncryption fromPb(StorageObject.CustomerEncryption customerEncryp
     /** Sets the blob's storage class. */
     public abstract Builder setStorageClass(StorageClass storageClass);
 
+    abstract Builder setTimeStorageClassUpdated(Long timeStorageClassUpdated);
```

Comment:
Follow the same pattern in custom time to raise an exception and implement this method to prevent breaking changes.

Reply:
done